File: src/test/java/no/hvl/tk/visual/debugger/settings/DebuggingVisualizerOptionTest.java
package no.hvl.tk.visual.debugger.settings;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.MatcherAssert.assertThat;

import org.junit.jupiter.api.Test;

class DebuggingVisualizerOptionTest {

  @Test
  void testToString() {
    assertThat(
        DebuggingVisualizerOption.EMBEDDED.toString(), is("Embedded visualizer (no interaction)"));
    assertThat(DebuggingVisualizerOption.WEB_UI.toString(), is("Browser visualizer"));
  }
}

File: src/test/java/no/hvl/tk/visual/debugger/settings/DebuggingVisualizerOptionTest.kt
package no.hvl.tk.visual.debugger.settings

import org.hamcrest.CoreMatchers
import org.hamcrest.MatcherAssert
import org.junit.jupiter.api.Test

internal class DebuggingVisualizerOptionTest {
  @Test
  fun testToString() {
    MatcherAssert.assertThat(
        DebuggingVisualizerOption.EMBEDDED.toString(),
        CoreMatchers.`is`("Embedded visualizer (no interaction)"))
    MatcherAssert.assertThat(
        DebuggingVisualizerOption.WEB_UI.toString(), CoreMatchers.`is`("Browser visualizer"))
  }
}
